test/tests/functional/pbs_hook_set_interrupt.py

```diff
@@ -68,14 +68,14 @@ def test_hook_interrupt(self):
         st = time.time()
         jid = self.server.submit(j)
 
-        self.server.log_match("TestHook Started", starttime=st, max_attempts=5)
+        self.server.log_match("TestHook Started", starttime=st)
 
         _msg = "Not Running: PBS Error: request rejected"
         _msg += " as filter hook '%s' got an alarm call." % hook_name
         _msg += " Please inform Admin"
         self.server.expect(JOB,
                            {'job_state': 'Q', 'comment': _msg},
-                           id=jid, offset=5, max_attempts=5)
+                           id=jid, offset=5)
```

Comment:
Can you combine the above 2 lines without going over 79 characters?

Reply:
No, line 77 is already at 63.  It would go over 79 char by a few characters.

Reply:
Thanks for checking